### PR TITLE
835239-UG document for the spline range area series in .NET MAUI 

### PR DESCRIPTION
**Pull request by EzhilarasanElangovan31**
**Description** 

Prepared UG document for the spline range area series in .NET MAUI

### CODE REVIEW

File: MAUI/Cartesian-Charts/SplineRangeArea.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+keywords: .net maui spline range area chart, maui spline range area chart, spline range area chart customization .net maui, syncfusion maui spline range area chart, cartesian spline range area chart maui, .net maui chart spline range area visualization.
+---
+
+# Spline Range Area Chart in .NET MAUI Chart
+
+## Spline Range Area Chart
+
+Spline Range Area Chart is used to visualize data points with smooth curves. In this series, the area between the curves is filled to indicate a range of values, such as a high and low price range or an upper and lower limit.
+
+To render a spline range area chart, create an instance of [SplineRangeAreaSeries]() and add it to the [Series](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCartesianChart.html#Syncfusion_Maui_Charts_SfCartesianChart_Series) collection property of [SfCartesianChart](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCartesianChart.html?tabs=tabid-1).
```
Comment (VimalaThirumalaiKumarSF3739) [inline MAUI/Cartesian-Charts/SplineRangeArea.md:17]: @EzhilarasanElangovan31 Update SfCartesianChart API link without tabs

File: MAUI/Cartesian-Charts/SplineRangeArea.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    Type = SplineType.Cardinal
+};
+
+chart.Series.Add(series);
+
+this.Content = chart;
+
+{% endhighlight %}
+
+{% endtabs %}
```
Comment (VimalaThirumalaiKumarSF3739) [inline MAUI/Cartesian-Charts/SplineRangeArea.md:133]: @EzhilarasanElangovan31 Need to include image for Type API also

File: MAUI/Cartesian-Charts/SplineRangeArea.md (hunk 1 above)
Comment (EzhilarasanElangovan31) [reply MAUI/Cartesian-Charts/SplineRangeArea.md]: Removed tabs

File: MAUI/Cartesian-Charts/SplineRangeArea.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+![Spline types in MAUI Chart]()
```
Comment (VimalaThirumalaiKumarSF3739) [inline MAUI/Cartesian-Charts/SplineRangeArea.md:135]: @EzhilarasanElangovan31 update as  - Spline types in MAUI Spline range area chart Chart

**Comment by VimalaThirumalaiKumarSF3739** [comment]
@EzhilarasanElangovan31 resolve CI failure

File: MAUI/Cartesian-Charts/SplineRangeArea.md (hunk 3 above)
Comment (EzhilarasanElangovan31) [reply MAUI/Cartesian-Charts/SplineRangeArea.md]: Updated

**Comment by VimalaThirumalaiKumarSF3739** [comment]
Fine

**Pull request merged**